scripts/webui.py

```diff
@@ -2444,6 +3707,9 @@ def run_RealESRGAN(image, model_name: str):
     'height': 512,
     'width': 512,
     'fp': None,
+    'mask_blur_strength': 1,
```

Comment:
What is the purpose of these changes?

Reply:
For those two txt2img and img2img arguments the defaults where missing in txt2img_defaults and img2img_defaults.
I added the missing values because, otherwise their defaults are unknown, because set from the frontend only. 
For scn2img I use txt2img_defaults and the other one to populate not specified arguments in the txt2img, etc. calls.
I wanted the used defaults values to be the same as the user would expect from using the txt2img and img2img web frontend. When they are missing the calls would have use the default values in txt2img/img2img argument list, or fail and throw an exception when there is no default value specified.